parsec/backend/asgi/logger.py

```diff
@@ -0,0 +1,51 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+from hypercorn.logging import Logger, AccessLogAtoms
+from typing import TYPE_CHECKING, Mapping, Iterable, Optional, Tuple, Union
+
+import base64
+import binascii
+
+if TYPE_CHECKING:
+    from hypercorn.config import Config
+    from hypercorn.typing import ResponseSummary, WWWScope
+
+
+class ParsecLogger(Logger):
+    def __init__(self, config: "Config") -> None:
```

Comment:
no need to overload this `__init__` given it just calls the parent one

Reply:
Done.